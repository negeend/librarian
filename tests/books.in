BOOK 111111
book 111111
BOOK HISTORY 111111
book history 111111
BOOK 111111 LONG
book 111111 long
ADD COLLECTION books.csv
BOOK 111111
BOOK 111111 LONG
book 111111
book 111111 long
BOOK 123456
BOOK 123456 LONG
BOOK HISTORY 111111
book history 111111
ADD MEMBER Fonald Blover
ADD MEMBER Brank Pocean
ADD MEMBER Jendrick Namar
RENT 100000 111111
RENT 100000 111112
RELINQUISH ALL 100000
RENT 100001 111111
RENT 100002 111112
RELINQUISH ALL 100001
RELINQUISH ALL 100002
RENT 100000 111112
RENT 100001 111111
RELINQUISH ALL 100000
RELINQUISH ALL 100001
BOOK HISTORY 111111
BOOK HISTORY 111112
RENT 100002 111111
BOOK HISTORY 111111
EXIT