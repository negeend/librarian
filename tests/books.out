user: No books in system.

user: No books in system.

user: No such book in system.

user: No such book in system.

user: No books in system.

user: No books in system.

user: 17 books successfully added.

user: To Kill a Mockingbird (Harper Lee)

user: 111111: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

user: To Kill a Mockingbird (Harper Lee)

user: 111111: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

user: No such book in system.

user: No such book in system.

user: No rental history.

user: No rental history.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: 100000
100001
100001

user: 100000
100002
100000

user: Success.

user: 100000
100001
100001

user: Ending Library process.
